### PR TITLE
Remove references to non-existent files in caffe2/python

### PR DESCRIPTION
**Pull request by SethHWeidman**
Removed references to non-existed files in caffe2/python.

Detected the existence of these files because the code in the Tensorboard tutorial located [here](https://github.com/pytorch/pytorch/blob/master/docs/source/tensorboard.rst) did not run; when I fixed these imports, it ran.

### CODE REVIEW

**Review by soumith — APPROVED** [review]
ship if tests pass

File: caffe2/python/scope.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 
 import contextlib
 import threading
-from past.builtins import basestring
```
Comment (bddppq) [inline caffe2/python/scope.py]: there is usage of `basestring` in this same file
https://github.com/pytorch/pytorch/blob/226ff68475b2a7b85e2277e69c3a68430c41aca8/caffe2/python/scope.py#L37

**Comment by SethHWeidman** [comment]
Hey @orionr I know we want to take the `experimental` tag off `torch.utils.tensorboard`. The three lines of code I modified were preventing the Tensorboard code from running for me; without them the code runs; see [here](https://nbviewer.jupyter.org/github/SethHWeidman/pytorch-tutorial/blob/master/tutorials/04-utils/tensorboard_pytorch/Tensorboard%20-%20Official%20tutorial.ipynb). Is it possible that the tests that are failing here are old tests that are no longer relevant?

Let me know if there's anything I can do to help here. Thanks.

**Comment by orionr** [comment]
If this is an issue, it looks like we need to scope down the import of Caffe2 inside `torch.utils.tensorboard`. So is the problem here that people need to `pip install future` for things to work without these changes?

**Comment by SethHWeidman** [comment]
@orionr the `torch.utils.tensorboard` code now works with the latest `master` branch of `pytorch`. Must have been a temporary bug introduced that caused this to fail. Could reveal a gap in our test coverage related to Tensorboard.

To @bddppq's point the fix I came up with must have had downstream impacts that were causing the tests to fail, so we shouldn't merge this. I'll close; sorry about the mix-up!

**Pull request closed without merging**